### DISCUSSION TITLE
amamba/intro/what/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1739] [Announcements]
# amamba/intro/what/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/intro/what/

**Comment by liyuerich** [comment]
export VERSION=**** # 修改为实际部署的版本。
helm repo add mspider-release https://release.daocloud.io/chartrepo/amamba
helm repo update amamba
helm upgrade --install --create-namespace --cleanup-on-fail amamba amamba-release/amamba -n amamba-system --version=${VERSION}


为什么 add mspider-release 使用的名字为mspider？
